test/overall_test.py

```diff
@@ -208,6 +208,22 @@
 >>> del(g)
 >>> del g_.data1
 >>> del(g_)
+>>> g1 = GenerativeImage()
+>>> function1 = eval("lambda x,y:" + g1.function1_str)
+>>> function2 = eval("lambda x,y:" + g1.function2_str)
```

Comment:
```
>>> function1 = eval("lambda x, y:" + g1.function1_str)
>>> function2 = eval("lambda x, y:" + g1.function2_str)
```

Reply:
fixed in https://github.com/sepandhaghighi/samila/pull/126/commits/c9b14e1ac8f8533be3917348d7105e3cedd9a19f